pages/linux/qm-unlock.md

```diff
@@ -0,0 +1,8 @@
+# qm unlock
+
+> Unlock virtual machines in QEMU/KVM Virtual Machine Manager.
```

Comment:
```suggestion
> Unlock a virtual machine in QEMU/KVM Virtual Machine Manager.
```
I think this would be better.

Reply:
Yeah, I think that looks better as well actually, especially because qm doesn't accept multiple values for unlock, so it may lead to confusion to use the plural form. Thanks for the suggestion!